experimental/displaynames/src/options.rs

```diff
@@ -35,6 +35,8 @@ pub struct DisplayNamesOptions {
     pub fallback: Fallback,
     /// The language display kind, defaults to "dialect".
     pub language_display: LanguageDisplay,
+    /// The option to retrieve the capitalization context setting.
+    pub capitalization: Capitalization,
```

Comment:
Suggestion: When implementing the capitalization, you should design it such that the actual capitalization functions can be provided by the client, since they may be data-driven.

I suggest waiting for a couple more weeks until icu_casemapping is more stable before working on this.